getting-started/exploring-forge.md

```diff
@@ -31,8 +31,8 @@ and, failing that, please [contact us][contact-timescale].
 1. First, supply your service name (e.g., `acmecorp-test` or `acmecorp-dev`).
 1. Next, choose your CPU and memory configuration, from (0.5 CPU, 2GB RAM) to 
 (8 CPU, 32 GB RAM).
-1. Select your storage requirements, from 25 GB to 1 TB.  Note that with TimescaleDB 
-compression, this is typically equivalent to 400 GB to 16+ TB of uncompressed 
+1. Select your storage requirements, from 25 GB to 4 TB.  Note that with TimescaleDB 
+compression, this is typically equivalent to 400 GB to 70+ TB of uncompressed 
```

Comment:
if we're going with 16x, should this be `64TB`?

Reply:
based on what we show in the UI it's 16.something which gets to about 67TB. That's why I went for 70+